cmd/fetch.go

```diff
@@ -24,7 +29,24 @@ results will also be inserted into a local SQLite database.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		name := args[0] // COT3100 or N00474503 etc.
 		isProfessor, _ := regexp.MatchString("N\\d{8}", name)
-
+		// Professor name is present
+		fmt.Println(isProfessor)
+		if !isProfessor && len(name) > 7 {
+			url := "https://webapps.unf.edu/faculty/bio/api/v1/faculty?searchLimit=1&searchTerm="
+			name = strings.ReplaceAll(name, " ", "%20") // Replaces space with %20 for web request
+			url += name
```

Comment:
Could we use string interpolation here? It avoids mutating the `name` variable:
```suggestion
			url :=  fmt.Sprintf(
					"https://webapps.unf.edu/faculty/bio/api/v1/faculty?searchLimit=1&searchTerm=%v",
			 		strings.ReplaceAll(name, " ", "%20")) // Replaces space with %20 for web request
```

Reply:
Made the changes